### PR TITLE
Editdocs

### PR DESCRIPTION
**Pull request by iregina**
Edited first 3 pages

### CODE REVIEW

File: pages/en/snapps/how-to-write-a-zkapp.mdx [hunk 1]
```diff
@@ -201,19 +201,19 @@ In the current design, the state can consist of at most 8 Fields of 32 bytes eac
...
+It takes the zkApp account address (a public key) as its only argument:
 
 ```ts
-let snappKey = PrivateKey.random();
-let snappAddress = PublicKey.fromPrivateKey(snappKey);
+let zkAppKey = PrivateKey.random();
+let zkAppAddress = PublicKey.fromPrivateKey(zkAppKey);
 
-let snapp = new SmartContract(snappAddress);
+let zkApp = new SmartContract(zkAppAddress);
```
Comment (jasongitmail) [inline pages/en/snapps/how-to-write-a-zkapp.mdx:210]: Let's come back and update this.

File: public/_redirects [hunk 2]
```diff
@@ -11,7 +11,7 @@
 /advanced/*                  /en/advanced/:splat
 /architecture/*              /en/architecture/:splat
 /developers/*                /en/developers/:spalt
-/snapps/*                    /en/snapps/:splat
```
Comment (MartinMinkov) [inline public/_redirects:14]: Would also be great to redirect `/snapps/*` -> `/zkapps` as well :D

File: pages/en/snapps/index.mdx [hunk 3]
```diff
@@ -1,48 +1,48 @@
...
 
 <br />
-<img src="/static/img/docs-images/CLIs.png" width="50%" />
+<img src="/static/img/docs-images/snapp-credit/zkcodesnippet.png" width="50%" />
 
-### What are snapps?
+### What are zkApps?
 
-Snapps (“snark apps”) are Mina’s **zk-SNARK**-powered smart contracts.
+zkApps (“snark apps”) are Mina’s **zk-SNARK**-powered smart contracts.
```
Comment (jasongitmail) [inline pages/en/snapps/index.mdx:19]: "zero-knowledge apps"

File: pages/en/snapps/how-to-write-a-zkapp.mdx (hunk 1 above)
Comment (jasongitmail) [reply pages/en/snapps/how-to-write-a-zkapp.mdx]:
```js
const privateKey = PrivateKey.random();
const address = PublicKey.fromPrivateKey(privateKey);

const contract = new SmartContract(address);
```

**Pull request merged**